bin/latexml

```diff
@@ -165,6 +165,7 @@ $serialized = undef;
 
 CheckDebuggable();
 
+UseSTDERR(undef);
```

Comment:
presumably latexmlpost deserves one of these too

Reply:
done!